vendor/composer/ClassLoader.php

```diff
@@ -279,7 +279,7 @@ public function isClassMapAuthoritative()
      */
     public function setApcuPrefix($apcuPrefix)
     {
-        $this->apcuPrefix = function_exists('apcu_fetch') && ini_get('apc.enabled') ? $apcuPrefix : null;
+        $this->apcuPrefix = function_exists('apcu_fetch') && filter_var(ini_get('apc.enabled'), FILTER_VALIDATE_BOOLEAN) ? $apcuPrefix : null;
```

Comment:
Is this auto-generated ? Or custom fixes ?

Reply:
Yes, that was generated by Composer.

Reply:
It was:

> Composer version 1.8.0 2018-12-03 10:31:16